.github/workflows/helm.yaml

```diff
@@ -4,41 +4,28 @@ name: Publish Helm
 # chart repo (that is to say it is released without further ado)
 env: {}
 on:
-    push:
-        branches: [main]
-        paths: ['charts/substra-frontend/**']
-    workflow_dispatch: {} # For running by hand from the web UI
-jobs:
-    build-helm:
-        name: Package Helm chart
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v4
-            - uses: azure/setup-helm@v3.5
-              with:
-                  version: 'v3.5.0'
-              id: install
-            - run: helm lint charts/substra-frontend
-
-            - name: Package chart
-              run: |
-                  helm package charts/substra-frontend
+  push:
+    branches:
+      - "main"
+    paths:
+      - "charts/substra-frontend/**"
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - "charts/substra-frontend/Chart.yaml"
+      - "charts/substra-frontend/values.yaml"
+      - "charts/substra-frontend/templates/**"
```

Comment:
Temporary bypasses the triggers because of changelog update. Will be reverted in: https://github.com/Substra/substra-frontend/pull/308